import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,173 @@
+	output, err := base.Execute(t, "export", "policy", "rate-limiting", "-n", name, "-e", env, "-k", "--verbose")
+	return output, err
+}
+
+// Imports Throttling policy to an env
+func importThrottlePolicy(t *testing.T, args *ThrottlePolicyImportExportTestArgs) (string, error) {
+	var output string
+	var err error
+	if args.Update {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath, "-u")
+	} else {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath)
+	}
+	return output, err
+}
+
+// Retrieve Throttling policy by name
+func getThrottlingPolicyByName(t *testing.T, args *ThrottlePolicyImportExportTestArgs, throttlePolicyName, policyType string) (map[string]interface{}, error) {
+	client := args.DestAPIM
+	uuid := client.GetThrottlePolicyID(t, args.Admin.Username, args.Admin.Password, throttlePolicyName, policyType)
+	policy := client.GetThrottlePolicy(uuid, policyType)
+	client.DeleteThrottlePolicy(uuid, policyType)
+	return ThrottlePolicyStructToMap(policy)
+}
+
+// Validates whether throttling policies are equal
+func ValidatePoliciesEqual(t *testing.T, args *ThrottlePolicyImportExportTestArgs, importedPolicy map[string]interface{}) {
+	exportedPolicy := args.Policy
+	exportedPolicy[PolicyIDKey] = PolicyIDReset
+	importedPolicy[PolicyIDKey] = PolicyIDReset
+	assert.Equal(t, exportedPolicy, importedPolicy)
+}
+
+// Converts Throttling policy struct to map
+func ThrottlePolicyStructToMap(policy interface{}) (map[string]interface{}, error) {
+	var throttlePolicy map[string]interface{}
+	marshalled, _ := json.Marshal(policy)
+	err := json.Unmarshal(marshalled, &throttlePolicy)
+	return throttlePolicy, err
+}
+
+// Cleanup func for exported throttling policy file
+func RemoveExportedThrottlingPolicyFile(t *testing.T, file string) {
+	t.Log("base.RemoveExportedThrottlingPolicyFile() - file path:", file)
+	if _, err := os.Stat(file); err == nil {
+		err := os.Remove(file)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+// Validates whether the throttling policy list is complete
+func ValidateThrottlePoliciesList(t *testing.T, jsonPretty bool, args *ThrottlePolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	base.WaitForIndexing()
+
+	output, _ := listThrottlePolicies(t, jsonPretty, args)
+
+	throttlePoliciesList := args.SrcAPIM.GetThrottlePolicies(t, args.CtlUser.Username, args.CtlUser.Password)
+
+	ValidateListThrottlePoliciesEqual(t, output, throttlePoliciesList)
+}
+
+// get the Throttle policy list apictl output
+func listThrottlePolicies(t *testing.T, JsonPretty bool, args *ThrottlePolicyImportExportTestArgs) (string, error) {
```

Comment:
Do we need to have this variable started in uppercase? Since this is not globally accessible, we should start with a lowercase letter [1].
Also, let's rename this something like "doJsonPrettyFormatting" or something to give the "boolean" meaning to the variable. (Do this renaming in all the places)

[1] https://stackoverflow.com/a/38616867/8095979

Reply:
fixed with 73381a57ed995c647ea2aa52394a7982f008363c